### ISSUE TITLE
Pie Chart has no width and height

### ISSUE THREAD

**Issue by PedroDaumas** [reflex-dev/reflex#1966]
**Describe the bug**
Whenever you create a piechart like the [example from the docs](https://reflex.dev/docs/library/graphing/piechart/), they chart has no width and height.
Adding variable widths and height also don't work. ie: width="20vw", height="20vh"

**To Reproduce**
 - Code/Link to Repo:
```
data = rx.recharts.pie_chart(
      rx.recharts.pie(
          data=[{"name": "test1", "value": 1}, {"name": "test2", "value": 2}],
          data_key="value",
          name_key="name",
          cx="50%",
          cy="50%",
          fill="#8884d8",
          width="20vw",
          height="20vh",
      ),
      rx.recharts.graphing_tooltip(),
      width="20vw",
      height="20vh",
  )

rx.box(
        data,
        width="20vw",
        height="20vh",
    )
```

**Expected behavior**
The graph should show up and be responsive.

**Screenshots**

**Specifics (please complete the following information):**
 - Python Version: 3.8
 - Reflex Version: 0.2.9
 - OS: linux
 - Browser (Optional):
 
**Additional context**
I managed to make it work using a fixed value for height and width but it's not responsive.

```
rx.recharts.pie_chart(
            rx.recharts.pie(
                data=[{"name": "test1", "value": 1}, {"name": "test2", "value": 2}],
                data_key="value",
                cx="50%",
                cy="50%",
                fill="#8884d8",
            ),
            rx.recharts.graphing_tooltip(),
            width=200,
            height=200,
)
```

**Comment by masenf** [comment]
Fixed by #1975 (next release 0.3.0)

**Comment by masenf** [comment]
> Adding variable widths and height also don't work. ie: width="20vw", height="20vh"

These css unit vars need to be set on an `rx.box` outside of the chart

```python
rx.box(
    rx.recharts.pie_chart(
            rx.recharts.pie(
                data=[{"name": "test1", "value": 1}, {"name": "test2", "value": 2}],
                data_key="value",
                cx="50%",
                cy="50%",
                fill="#8884d8",
            ),
            rx.recharts.graphing_tooltip(),
    ),
    width="20vw",
    height="20vh",
)
